pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -522,15 +537,22 @@ class Device(BaseDevice):
         default_noise_model: An optional noise model characterizing the default
             noise of the device. Can be used by emulator backends that support
             noise.
+        requires_layout: Whether the register used in the sequence must be
+            created from a register layout. Only enforced in QPU execution.
         pre_calibrated_layouts: RegisterLayout instances that are already
             available on the Device.
+        accepts_new_layouts: Whether registers built from register layouts
+            that are not already calibrated are accepted. Only enforced in
+            QPU execution.
     """
 
     max_atom_num: int
     max_radial_distance: int
+    requires_layout: bool = True
     pre_calibrated_layouts: tuple[RegisterLayout, ...] = field(
         default_factory=tuple
     )
+    accepts_new_layouts: bool = True
```

Comment:
Very Nit: Should this be True or False by default ? Current QPUs would be False...

Reply:
What happens if you don't require a layout but don't accept new layouts ? Should it be prohibited here ?

Reply:
Okay let's go with the least prohibitive default ! I was wondering if we should go with the least prohibitive or the most prohibitive (since it real device). Sounds good to me !